test-files/cases/attribute/unifying/highway-2927-name-preservation-1/Expected.osm

```diff
@@ -1,351 +1,351 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <osm version="0.6" generator="hootenanny" srs="+epsg:4326">
-    <bounds minlat="10.5053" minlon="-66.91539999999999" maxlat="10.509974" maxlon="-66.9121"/>
-    <node visible="true" id="-87" timestamp="2019-01-17T19:50:06Z" version="1" changeset="841" lat="10.5065750463800001" lon="-66.9152038258099822">
+    <bounds minlat="10.50529998962702" minlon="-66.91539999999999" maxlat="10.50997398962358" maxlon="-66.9121"/>
+    <node visible="true" id="-87" timestamp="2019-01-17T19:50:06Z" version="1" changeset="841" lat="10.5065750360060814" lon="-66.9152038258099822">
         <tag k="hoot:status" v="1"/>
         <tag k="hoot:id" v="-87"/>
     </node>
-    <node visible="true" id="-86" timestamp="2019-01-17T19:50:06Z" version="1" changeset="841" lat="10.5065369230400005" lon="-66.9150022248399949">
+    <node visible="true" id="-86" timestamp="2019-01-17T19:50:06Z" version="1" changeset="841" lat="10.5065369126661086" lon="-66.9150022248399949">
```

Comment:
These coordinates are very different from each other.  I assume you've visually inspected these files and compared them in JOSM?

Reply:
`hoot diff` declares the files as being the same. Looking in JOSM some roads have shifted by about 2mm.

Reply:
2mm is good enough for me!